ansibullbot/triagers/defaulttriager.py

```diff
@@ -118,6 +118,20 @@ def __init__(self, args=None):
             server=C.DEFAULT_GITHUB_URL
         )
 
+        self._maintainer_team = None
+
+    @property
+    def maintainer_team(self):
+        # Note: this assumes that the token used by the bot has access to check
+        # team privileges across potentially more than one organization
+        if self._maintainer_team is None:
+            self._maintainer_team = []
+            teams = C.DEFAULT_GITHUB_MAINTAINERS
+            for team in teams:
+                _org, _team = team.split('/')
+                self._maintainer_team.extend(self.ghw.get_members(_org, _team))
+        return list(set(sorted([x for x in self._maintainer_team if x not in C.DEFAULT_BOT_NAMES])))
```

Comment:
I would move the `list(set(sorted...` processing to the above `if` branch as well.

Also I am curious why passing `sorted` into `set` since `list` makes it unsorted again? Would the following make more sense?
```python
sorted(set([x for x in self._maintainer_team if x not in C.DEFAULT_BOT_NAMES]))
```

Or just do:
```python
sorted(set(self._maintainer_team).difference(C.DEFAULT_BOT_NAMES)))
```